README.md

```diff
@@ -64,6 +64,68 @@ render(
 
 For more examples, see the `example/` directory.
 
+## Usage with redux dev tools
+
+Install the browser plugin available in [redux dev tools](http://extension.remotedev.io/), follow the instructions in the previous link.
+
+Once done, the only change you need do in order to use with redux dev tools is a little change in the `Container` classes, in the example bellow we assign the "Counter" name for the redux dev tools widget.
+
+```js
+// @flow
+import { Container } from 'unstated';
+
+type CounterState = {
+  count: number
+};
+
+class CounterContainer extends Container<CounterState> {
+  state = {
+    count: 0
+  };
+
+  // Widget name to show in the redux devl tools panel
+  name = 'Counter';
+
+  increment() {
+    // With no info, backward compatibility
+    this.setState({ count: this.state.count + 1 });
+
+    // With no info and providing a callback to be run after the state changes
+    // backward compatibility
+    this.setState({ count: this.state.count + 1 }, () => {
+      console.log('Counter after state changes: ' + this.state.count);
+    });
+
+    // With custom info for redux dev tools
+    this.setState({ count: this.state.count + 1 }, 'INCREMENT');
```

Comment:
I don't really like the use of a new argument. I would rather something like getting the caller and source location like:

```js
function setState(state) {
  let err = new Error('message');
  let name = err.constructor.name;
  let message = err.message;
  let prefix = `${name}: ${message}\n`;
  let stack = err.stack.replace(prefix, '');
  let lines = stack.split('\n').map(line => line.trim());
  let caller = lines[1];
  
  console.log(caller);
}

function action() {
  setState({ a: 1, b: 2 });
}

action();
```